### PR TITLE
Feat/convert call to contribute chakra

### PR DESCRIPTION
**Pull request by vdusart**
Convert the CallToContribute component to Chakra

# Related Issue

Epic: https://github.com/ethereum/ethereum-org-website/issues/6374

### CODE REVIEW

File: src/components/CallToContribute.tsx [hunk 1]
```diff
@@ -1,82 +1,36 @@
...
-`
-
-const Title = styled.h2`
-  font-family: ${(props) => props.theme.fonts.monospace};
-  text-transform: uppercase;
-  background: ${(props) => props.theme.colors.border};
-  padding: 0.25rem;
-`
+import { Flex, Heading, Icon, Text } from "@chakra-ui/react"
+import { FaGithub } from "react-icons/fa"
```
Comment (pettinarip) [inline src/components/CallToContribute.tsx:7]: Import order. External libs should be at the top of the list.

File: src/components/CallToContribute.tsx [hunk 2]
```diff
@@ -1,82 +1,36 @@
...
 export interface IProps {
   editPath: string
 }
 
 const CallToContribute: React.FC<IProps> = ({ editPath }) => (
-  <StyledCard>
-    <ImageColumn>
+  <Flex
+    bg="ednBackground"
+    alignItems="center"
```
Comment (pettinarip) [inline src/components/CallToContribute.tsx:16]:
```suggestion
    align="center"
```

File: src/components/CallToContribute.tsx [hunk 3]
```diff
@@ -1,82 +1,36 @@
...
+    mt={8}
+    borderWidth="1px"
+    borderStyle="solid"
```
Comment (pettinarip) [inline src/components/CallToContribute.tsx:19]: This is the same as these two lines
```suggestion
    border="1px"
```

File: src/components/CallToContribute.tsx [hunk 4]
```diff
@@ -87,37 +41,78 @@ const CallToContribute: React.FC<IProps> = ({ editPath }) => (
...
+      <Text lineHeight="140%" color="text" fontFamily="monospace">
         <Translation id="page-calltocontribute-desc-4" />{" "}
         <Link to="https://discord.gg/CetY6Y4">
           <Translation id="page-calltocontribute-link-2" />
         </Link>{" "}
-      </Description>
-      <GithubButton to={editPath}>
-        <GithubIcon name="github" />{" "}
+      </Text>
+      <Flex
```
Comment (pettinarip) [inline src/components/CallToContribute.tsx:83]: Lets refactor this button like this:
```tsx
<ButtonLink to={editPath} leftIcon={<Icon as={FaGithub} />}>
```

File: src/components/CallToContribute.tsx [hunk 5]
```diff
@@ -87,37 +41,78 @@ const CallToContribute: React.FC<IProps> = ({ editPath }) => (
...
+      >
+        <Icon
+          fill="background"
+          mr={2}
+          w={6}
+          h={6}
+          as={FaGithub}
+          name="github"
+        />{" "}
         <span>
```
Comment (pettinarip) [inline src/components/CallToContribute.tsx:110]: With the previous refactor this `span` should be removed.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Great job @vdusart left a few change requests.

File: src/components/CallToContribute.tsx [hunk 6]
```diff
@@ -87,37 +40,71 @@ const CallToContribute: React.FC<IProps> = ({ editPath }) => (
...
-        <GithubIcon name="github" />{" "}
-        <span>
-          <Translation id="page-calltocontribute-span" />
-        </span>
-      </GithubButton>
-    </Column>
-  </StyledCard>
+      </Text>
+      <ButtonLink
+        as={ButtonLink}
```
Comment (vdusart) [inline src/components/CallToContribute.tsx:83]: If I remove this line the background style is ignored :confused:

Comment (pettinarip) [reply src/components/CallToContribute.tsx:83]: Yes, lets remove all those custom style props we have in this particular button.

With `<ButtonLink to={editPath} leftIcon={<Icon as={FaGithub} />}>` we should have almost the same styles.

Comment (pettinarip) [reply src/components/CallToContribute.tsx:83]: I think it is a good opportunity to clean this button and leave the standard solid styled button we are using from now on.

Comment (vdusart) [reply src/components/CallToContribute.tsx]: Ok let's do that !

**Review by pettinarip — APPROVED** [review]
Greaaat! 🚀 @vdusart thanks man

**Pull request merged**